src/app/inventory/item-move-service.ts

```diff
@@ -566,7 +566,7 @@ function chooseMoveAsideItem(
     throw new DimError(
       'no-space',
       t('ItemService.NotEnoughRoom', { store: target.name, itemname: item.name })
-    );
+    ).withError(new DimError('ItemService.NotEnoughRoomGeneral'));
```

Comment:
Why not just have a distinct error that represents these two things? Triggering a "cause" error makes more sense to me if it was thrown from somewhere else and caught here.

Reply:
Take a look at how the error grouping works - the error specifically calls out the item, but the errors need something to be grouped on, and I used the cause as a grouping key. LMK if you have a better idea.